pkg/cluster/nodes/node.go

```diff
@@ -108,16 +111,16 @@ func tryUntil(until time.Time, try func() bool) bool {
 	return false
 }
 
-// LoadImages loads image tarballs stored on the node into docker on the node
+// LoadImages loads image tarballs stored on the node into container storage on the node
 func (n *Node) LoadImages() {
-	// load images cached on the node into docker
+	// load images cached on the node into Container Storage
 	if err := n.Command(
 		"find",
 		"/kind/images",
 		"-name", "*.tar",
-		"-exec", "docker", "load", "-i", "{}", ";",
+		"-exec", container.Engine, "load", "-i", "{}", ";",
```

Comment:
this is calling docker inside the "node" container, so this should still be `"docker"` for now since that's still what runs inside.

Reply:
Ok I will change it back, but I actually think podman inside of that container would make more sense since you don't need ot setup a separate daemon, or is this container volume mounting in the docker.sock?

Reply:
podman does look very cool for that, however:
- We're using docker for now because of existing expertise and accesibility to both our maintainers and the bulk of our known current and potential users. 
- I also need to maintain another more general docker-in-docker setup for [kubernetes/test-infra](https://github.com/kubernetes/test-infra) where it is used pervasively by many projects we merely support and that is less likely to change, as we'd need to convince them all to switch. @munnerz also maintains a parallel test-infra setup to this one.

It's certainly worth looking at, but I don't think the second point would happen and there is very large overlap in maintaining and debugging these setups. There's also a _lot_ more information out there from third parties related to dind. I think @runcom might be the first person to run cri-o in a container :upside_down_face: 

We're not mounting sockets.

Other than loading the images, only Kubernetes is supposed to be talking to this, however some advanced users are already talking to it, mostly to load their images until #28.